reframe/frontend/executors/policies.py

```diff
@@ -94,13 +94,36 @@ def on_task_exit(self, task):
         pass
 
     def on_task_failure(self, task):
+        timings = task.pipeline_timings(['compile_complete',
+                                         'run_complete',
+                                         'total'])
+        msg = f'{task.check.info()} [{timings}]'
         if task.failed_stage == 'cleanup':
-            self.printer.status('ERROR', task.check.info(), just='right')
+            self.printer.status('ERROR', msg, just='right')
         else:
-            self.printer.status('FAIL', task.check.info(), just='right')
+            self.printer.status('FAIL', msg, just='right')
+
+        timings = task.pipeline_timings(['setup',
+                                         'compile_complete',
+                                         'run_complete',
+                                         'sanity',
+                                         'performance',
+                                         'total'])
+        getlogger().verbose(f"==> {timings}")
```

Comment:
A shortcut function like that would be better, similar to what you have proposed:

```suggestion
        getlogger().verbose(f"==> {task.pipeline_timings_all()}")
```